docs/README.md

```diff
@@ -323,7 +325,12 @@ node {
 
 ## Configuration
 
-The plugin's default behavior requires **no configuration**. If you need to change the configuration, you can use the Web UI or CasC.
+Two sources of configuration drive how the plugin works:
+
+- *AWS Java SDK configuration* (outside of Jenkins). Consult the AWS SDK documentation to find where to put the SDK configuration.
+- *Plugin configuration* (inside Jenkins). You can use the Jenkins Web UI or CasC to customise it.
```

Comment:
Need to be more clear since the WEB UI does not offer all the configuration required to get the plugin running on the first place. The WEB UI only have some further tuning.